src/test/java/plugins/DeployPluginTest.java

```diff
@@ -29,12 +29,12 @@
 public class DeployPluginTest extends AbstractJUnitTest {
 
     @Inject
-    DockerContainerHolder<Tomcat7Container> docker;
+    DockerContainerHolder<Tomcat10Container> docker;
 
     @Test
     @Native("bash")
     @WithCredentials(credentialType = WithCredentials.USERNAME_PASSWORD, values = {"admin", "tomcat"}, id = "tomcat")
-    public void deploy_sample_webapp_to_tomcat7() throws IOException, InterruptedException {
+    public void deploy_sample_webapp_to_tomcat10() throws IOException, InterruptedException {
```

Comment:
I do not believe deploying in tomcat 10 vs deploying in tomcat 7 changes the intent of the test